.travis.yml

```diff
@@ -1,18 +1,25 @@
 language: scala
-sudo: false
+branches:
+  except:
+  - release
+sudo: true
 scala:
-  - 2.11.12
-  - 2.12.6
+- 2.11.12
+- 2.12.6
 jdk:
-  - oraclejdk8
+- oraclejdk8
 before_cache:
-  - find $HOME/.ivy2 -name "ivydata-*.properties" -delete
-  - find $HOME/.sbt -name "*.lock" -delete
+- find $HOME/.ivy2 -name "ivydata-*.properties" -delete
+- find $HOME/.sbt -name "*.lock" -delete
 cache:
   directories:
-    - $HOME/.ivy2/cache
-    - $HOME/.sbt/boot/
+  - "$HOME/.ivy2/cache"
+  - "$HOME/.sbt/boot/"
 script:
   - sbt ++$TRAVIS_SCALA_VERSION validate
```

Comment:
I'd assume we should just toggle into `build.sh` here and we're done?

Reply:
assuming the build matrix stuff is set correctly, then yes